services/_system.md

```diff
@@ -91,7 +91,7 @@ Asks device to stream a given number of samples
 
 Period between packets of data when streaming in milliseconds.
 
-    ro reading: i32 @ 0x101
+    ro volatile reading: i32 @ 0x101
```

Comment:
reading is always volatile. i don't think we should require it.

Reply:
This is simply an annotation we provide in the system spec for information. We have the vocabulary "volatile" now - we should use it.  It's not a requirement - it's documentation.

Reply:
But all reading register will have to be marked with volatile?